modules/sharing-registry/sharing-data-migrator/src/main/java/org/apache/airavata/sharing/registry/migrator/airavata/AiravataDataMigrator.java

```diff
@@ -181,6 +189,34 @@ public static void main(String[] args) throws SQLException, ClassNotFoundExcepti
             }
         }
 
+        //Creating application entries
+        query = "SELECT * FROM APPLICATION";
+        statement = expCatConnection.createStatement();
+        rs = statement.executeQuery(query);
+        while(rs.next()){
+            try {
+                Entity entity = new Entity();
+                entity.setEntityId(rs.getString("APPLICATION_ID"));
+                entity.setDomainId(rs.getString("GATEWAY_ID"));
+                entity.setEntityTypeId(rs.getString("GATEWAY_ID") + ":APPLICATION");
+                entity.setOwnerId(rs.getString("USER_NAME") + "@" + rs.getString("GATEWAY_ID"));
+                entity.setParentEntityId(rs.getString("PROJECT_ID"));
```

Comment:
Let's just leave parent entity id null since there really isn't a parent of an application.

Reply:
Done.